.circleci/scripts/binary_populate_env.sh

```diff
@@ -156,7 +156,7 @@ EOL
 # nproc doesn't exist on darwin
 if [[ "$(uname)" != Darwin ]]; then
   # Because most Circle executors only have 20 CPUs, using more causes OOMs w/ Ninja and nvcc parallelization
-  MEMORY_LIMIT_MAX_JOBS=18
+  MEMORY_LIMIT_MAX_JOBS=12
```

Comment:
For posterity, please add a comment on why MAX_JOBS is set as such.  Otherwise, the context would be lost after a while :P